EIT/EIT.py

```diff
@@ -137,64 +159,71 @@ def Heaviside(func):
 
 # creating PDE forms
 PDE_form1 = (form_lhs, form_rhs1)
-PDE_form2 = (form_lhs, form_rhs2)
-PDE_form3 = (form_lhs, form_rhs3)
-PDE_form4 = (form_lhs, form_rhs4)
-
 
 # creating PDE models
-# for the first PDE problems we specify to reuse the factorization of the lhs for the rest of the PDE modesl
-PDE1 = cuqipy_fenics.pde.SteadyStateLinearFEniCSPDE(PDE_form1, mesh, solution_space, parameter_space, zero_bc, observation_operator=observation1, reuse_assembled=True)
-# We copy the PDE1 for the rest of the PDE problems 
+# for the first PDE problems we specify to reuse the factorization of the lhs 
+# for the rest of the PDE models
+PDE1 = SteadyStateLinearFEniCSPDE(
```

Comment:
I suggest splitting to have a single input on each line to make all inputs appear equally important/visible. This makes it also visually simpler in my opinion.

Reply:
done ✅